src/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+rock_library(gps_ublox
+    SOURCES Driver.cpp protocols/UBX.cpp Driver.cpp
+    HEADERS Driver.hpp protocols/Parser.hpp protocols/UBX.hpp Driver.hpp
+    DEPS_PKGCONFIG iodrivers_base base-types)
+
+rock_executable(gps_ublox_cli Main.cpp
```

Comment:
I've taken the habit of using `_ctl` (for control). Since the IMU and other recent drivers followed that pattern, I'd rather that we stick to it.

Reply:
Agreed.. :+1: